HomebrewFormula/radicle-tools.rb

```diff
@@ -0,0 +1,31 @@
+class RadicleTools < Formula
+  version "0.0.0"
+  desc "Radicle CLI tools"
+  homepage "https://github.com/radicle-dev/radicle-client-tools"
+
+  url "https://github.com/radicle-dev/radicle-client-tools/releases/download/v#{version}/radicle-tools-x86_64-apple-darwin.tar.gz"
```

Comment:
We should start to think about how we could host binaries on radicle :thinking:

Reply:
Maybe.  That sounds like a task to be considered in its own right though, so maybe let's put it on a backlog?

Reply:
Aye, just thinking aloud